src/App.js

```diff
@@ -6,14 +6,18 @@ import Home from "./pages/Home";
 import Friends from "./pages/Friends";
 import BuddyProfile from "./pages/BuddyProfile";
 import LogIn from "./pages/LogIn";
+import ActivityShow from "./pages/ActivityShow";
+import users from "./mockUsers";
 
-function App() {
 
+function App() {
+const [activities, setActivities] = useState(users)
```

Comment:
Why are users being set to activities? Also please remove console log. Your routes need to be commented in if they are going to be used and capitalization matters. Need to be consistent, please use lowercase for all.